command/operator_generate_root.go

```diff
@@ -242,7 +242,16 @@ func (c *OperatorGenerateRootCommand) Run(args []string) int {
 	case c.flagGenerateOTP:
 		otp, code := c.generateOTP(client, kind)
 		if code == 0 {
-			return PrintRaw(c.UI, otp)
+			switch Format(c.UI) {
+			case "", "table":
```

Comment:
You can omit the `""` case, since `Format` already translates that to `"table"`.